python/tests/rtl/test_serialize.py

```diff
@@ -95,6 +95,7 @@ class Model(ml.Model):
     list_model_no_refs1: Sequence["ModelNoRefs1"]
     opt_enum1: Optional["Enum1"] = None
     opt_model_no_refs1: Optional["ModelNoRefs1"] = None
+    list_opt_model_no_refs1: Optional[Sequence["ModelNoRefs1"]] = None
```

Comment:
thought I saw some odd behavior so I added this version of type to make sure cattrs gets it right